main.go

```diff
@@ -146,8 +148,11 @@ func uiMain(driver gxui.Driver) {
 		workingDir = os.Getenv("HOME")
 	}
 	for _, file := range files {
-		filepath := filepath.Join(workingDir, file)
-		commander.Controller().Editor().Open(filepath, token.Position{})
+		path := file
+		if !globalSearch {
+			path = filepath.Join(workingDir, file)
+		}
```

Comment:
I think we can actually do this with [`filepath.Abs`](https://golang.org/pkg/path/filepath/#Abs):
```go
file = filepath.Abs(file)
```
The docs state that if the file path is not absolute, it will be joined with the working directory.  This eliminates the need for the `globalSearch` flag.